requirements.txt

```diff
@@ -32,9 +32,8 @@ sentencepiece>=0.2.0
 tokenizers>=0.11.1
 toml>=0.10.2
 # Default version. Can be overridden in extra requirements
-torch>=2.3.0,<2.6.0
+torch>=2.6.0,<2.7.0
```

Comment:
does it have to be capped now that it's pinned via constraints? Do we know 2.7.0+ is broken?

Reply:
Yeah, that's true. We can cap in the `constraints-dev.txt` and leave it uncapped here. I'll also leave a comment in this file letting people know to not put a cap here.